pkg/multiboot/description.go

```diff
@@ -36,7 +36,7 @@ type Description struct {
 
 // Description returns string representation of
 // multiboot information.
-func (m Multiboot) Description() (string, error) {
+func (m multiboot) Description() (string, error) {
```

Comment:
I think we can unexport this method, it is called only if the kernel loaded in debug/verbose mode.